prow/daily-e2e-cluster_wide-auth.sh

```diff
@@ -26,27 +26,42 @@ set -u
 # Print commands
 set -x
 
+# exports $HUB, $TAG, and ISTIOCTL_URL
 source greenBuild.VERSION
+echo "Using artifacts from HUB=${HUB} TAG=${TAG} ISTIOCTL_URL=${ISTIOCTL_URL}"
+
 ISTIO_SHA=`curl $ISTIOCTL_URL/../manifest.xml | grep istio/istio | cut -f 6 -d \"`
 [[ -z "${ISTIO_SHA}"  ]] && echo "error need to test with specific SHA" && exit 1
 
-echo "Using artifacts from HUB=${HUB} TAG=${TAG} ISTIOCTL_URL=${ISTIOCTL_URL}"
-
+mkdir -p ${GOPATH}/src/istio.io
+cd ${GOPATH}/src/istio.io
 git clone -n https://github.com/istio/istio.git
 cd istio
 git checkout $ISTIO_SHA
 
-
 source "./prow/cluster_lib.sh"
-
 trap delete_cluster EXIT
 create_cluster 'cluster-wide-auth'
 
+ISTIO_GO=$(cd $(dirname $0)/..; pwd)
+
+# Download envoy and go deps
+${ISTIO_GO}/bin/init.sh
+
+# Build istioctl, used by  the test.
+make depend.ensure istioctl generate_yaml
```

Comment:
not sure why you need depend.ensure

Reply:
thinking more both istioctl, generate_yaml should not be built again. you should download them and run the tests against the prebuilt yaml/istioctl